versions.tf

```
@@ -1,4 +1,11 @@
 
 terraform {
   required_version = ">= 0.12"
```

```suggestion
  required_version = ">= 0.13"
```
Your syntax below necessitates 0.13

---

I think it just requires the final version of 0.12, but requiring 0.13 shouldn't be a problem